test/bootstrap_radio_button_test.rb

```diff
@@ -14,7 +14,7 @@ class BootstrapRadioButtonTest < ActionView::TestCase
         </label>
       </div>
     HTML
-    assert_equivalent_xml expected, @builder.radio_button(:misc, '1', label: 'This is a radio button')
+    assert_with_builder_radio(expected, :misc, '1', label: 'This is a radio button')
```

Comment:
The fact that this test file has to change at all as a result of adding `form_with` indicates to me that there is a little too much going on in this PR. Adding one new feature (`bootstrap_form_with`) should mean adding one new set of tests. It should not require existing tests to be rewritten. I would prefer to see these tests left alone and the `assert_with_builder_radio` and `assert_with_builder` helpers excluded from this PR.

Reply:
I'll remove all testing for `bootstrap_form_with` and `fields` other than the two cases you mentioned above. I'll continue to work on my testing branch and see if I still feel there's a need for more testing.

I've come around to the idea of making this PR minimal, so that's what I'll do. FWIW, the reasons why I wrote more extensive tests are:

* If `form_with` is someday going to replace `form_for` and `form_tag`, then I thought it would be better write a complete set of `bootstrap_form_with` tests while I had my head in the problem, rather than leaving it to someone else, five years from now, to figure out how to migrate the `bootstrap_form_for` tests to `bootstrap_form_with`
* Back in August I understood very little of this gem, and wanted to make sure I wasn't breaking anything
* When I tried to use `bootstrap_form_with` in my app, lots of stuff broke. Most of it turned out to be due to the differences between `form_for` and `form_with`, but I didn't figure that out until after I wrote some tests for `bootstrap_form_with`
* General distrust of myself any time I think that, "I only need to test this one little difference". :-)

And I really appreciate the feedback, so keep the comments coming!